Formula/parity.rb

```diff
@@ -2,22 +2,22 @@
 
 class Parity < Formula
   homepage "https://github.com/thoughtbot/parity"
-  sha256 "d56827ce379958b5abe386f6afff3b012275f43a6d982f524c54bb8a790cee20"
-  url "https://github.com/thoughtbot/parity/releases/download/v0.9.2/parity-0.9.2-osx.tar.gz"
-  version "0.9.2.1"
+  sha256 "438b15beb0f0e0fd8da1aee5cc47ca57ff03c138091100d30fe5ae90a0f22c22"
+  url "https://github.com/thoughtbot/parity/releases/download/0.9.3/parity-0.9.3-osx.tar.gz"
+  version "0.9.3"
 
   depends_on "git"
   depends_on "heroku-toolbelt"
   depends_on "postgres"
 
   def install
-    prefix.install "lib" => "lib"
+    lib.install Dir["lib/*"]
 
     bin.install "bin/development", "bin/staging", "bin/production"
   end
 
   devel do
-    url "https://github.com/thoughtbot/parity/releases/download/v0.9.3.beta/parity-0.9.3.beta-osx.tar.gz"
+    url "https://github.com/thoughtbot/parity/releases/download/v0.9.3.beta/parity-0.9.3.beta2-osx.tar.gz"
```

Comment:
I created the development formula so that we could do testing without needing to repush the Homebrew formula over and over again. We don't have a 0.9.4 or 1.0.0 development version yet, so I'm not sure what to do here.